node/src/blockchain/blockchain_bridge.rs

```diff
@@ -721,8 +726,18 @@ mod tests {
             false,
         );
         subject.payable_payments_setup_subs_opt = Some(accountant_recipient);
+        let raw_qualified_payables = QualifiedPayablesRawPack {
+            payables: qualified_payables
+                .clone()
+                .into_iter()
+                .map(|payable| QualifiedPayablesBeforeGasPriceSelection {
+                    payable,
+                    previous_attempt_gas_price_minor_opt: None,
+                })
+                .collect(),
```

Comment:
I wonder if you use this more often, then maybe you should transform into a utility method.